### DISCUSSION TITLE
Running parallel steps in a job

### DISCUSSION THREAD

**Discussion by nihed-b** [community/community#26727] [Actions]
<p>Hello,<br>
This items was already discussed but no recent status, so I’m aiming to run parallel steps for the same job.<br>
Currently, parallelization could be made to run jobs in parallel for the same workflow.</p>
<p>Is it possible to run steps in parallel inside the same job?<br>
As for instance:<br>
step 1 and then step A and B in parallel and finally step 3 :<br>
<img src="https://user-images.githubusercontent.com/103523089/181099443-568c348b-233b-4a66-abac-01feda040e11.png" alt="parallel" data-base62-sha1="b57WpE4FE5LLyPOnXSZ3Up6Xgtk" width="550" height="182"></p>
<p>Thanks in advance</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="nihed-b" data-post="1" data-topic="244538">
<div class="title">
<div class="quote-controls"></div>
 nihed-b:</div>
<blockquote>
<p>Is it possible to run steps in parallel inside the same job?</p>
</blockquote>
</aside>
<p>Not at the step level. What you can do is start multiple processes in the background (so they’ll run in parallel) within one step, and wait for them to complete.</p>

**Comment by nihed-b** [comment]
<p>Thanks for the clarification :slight_smile:</p>

**Comment by qoomon** [comment]
Now you can run also steps in parallel feel free to try my action Now it's possible, feel free to try my parallel-steps action https://github.com/qoomon/actions--parallel-steps

@nihed-b
